velox/expression/tests/ArgumentTypeFuzzerTest.cpp

```diff
@@ -326,4 +326,55 @@ TEST_F(ArgumentTypeFuzzerTest, unconstrainedSignatureTemplate) {
   ASSERT_EQ(argumentTypes[0]->childAt(0), argumentTypes[1]);
 }
 
+TEST_F(ArgumentTypeFuzzerTest, orderableConstraint) {
```

Comment:
The change looks good, but I wonder how is this test verifying the new functionality. Did this test fail before the change?

Reply:
No, this test would not fail before this change. The return type reverse binding would fail too if there is an `orderableTypeVariable` constraint. It won't reach the code path of `determineUnboundedTypeVariables`, in which `randOrderatype()` is called with or without this change.
```C++
if (returnType_) {
    exec::ReverseSignatureBinder binder{signature_, returnType_};
    if (!binder.tryBind()) {
      return false;
    }
```

Reply:
@mbasmanova I've added the fellowing test block without return type reverse binding.
```C++
{
    for (int i = 0; i < 100; ++i) {
      auto signature = exec::FunctionSignatureBuilder()
                           .orderableTypeVariable("T")
                           .returnType("bigint")
                           .argumentType("T")
                           .build();
      std::mt19937 seed{123456};
      ArgumentTypeFuzzer fuzzer{*signature, nullptr, seed};
      fuzzer.fuzzArgumentTypes(kMaxVariadicArgs);
      ASSERT_TRUE(fuzzer.argumentTypes()[0]->isOrderable());
    }
  }
```